src/components/DateRangePicker.js

```diff
@@ -695,4 +704,4 @@ class DateRangePicker extends React.Component {
   };
 }
 
-module.exports = withTheme(Radium(DateRangePicker));
```

Comment:
And here is the second change. I've removed Radium in favor of using glamor in the related components.